### PR TITLE
Adding new optional uses for HTWHLSTC

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
Cooling and heating percentage commands controlled by supply_air_setpoint

### CODE REVIEW

**Comment by Ciaran-Oflannagain** [comment]
*temperature_setpoint

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -5861,6 +5861,8 @@ HTWHLSTC:
   - high_supply_air_temperature_alarm
   - low_return_air_temperature_alarm
   - low_supply_air_temperature_alarm
+  - heating_percentage_command
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:5864]: I'm not sure I understand how these are cooling/heating commands given to the heat wheel. The heat wheel can only modulate its rotational speed to transfer more or less energy between the supply and exhaust air streams. What do these fields mean in this context?

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Hi @cstirdivant - apologies for confusion. The heating and cooling commands are actually coils. These are a separate unit to the the heatwheel. The thermal wheel will heat supply air to a certain amount then the chilling/heating valves operate. I have changed the HTWHLSTC abstract type back and submitted the DOAS entity type. Thanks, Ciaran

**Comment by Ciaran-Oflannagain** [comment]
Hi @cstirdivant  - please see updated DOAS entity type instead of new HTWHLSTC abstract type at your convenience. Thanks, Ciaran

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml [hunk 2]
```diff
@@ -16,6 +16,23 @@
...
+  - HTWHLSTC
+  - SFVSC
+  - EFVSC
+  - EDM
+  - SDM
+  - RFC
+  - SFC
+  - FDPM2X
+  - RTM
+  - FPHC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/DOAS.yaml:34]: FPHC is typically used for radiant heating/cooling (radiant beams, panels, etc.) -- why not use CHWSC and HWSC ?

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/DOAS.yaml]: Thank you for input  - have used both CHWSC and HWSC to cover heating and cooling valve percentage commands

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**